pyproject.toml

```diff
@@ -26,7 +26,7 @@ classifiers = [
     "Programming Language :: Python :: 3.13", 
     "Topic :: Scientific/Engineering :: Physics",
 ]
-dependencies = ["pennylane>=0.40", "scipy-openblas32>0.3.26"]
+dependencies = ["pennylane>=0.40", "scipy-openblas32>=0.3.26"]
```

Comment:
```suggestion
dependencies = ["pennylane>=0.40", "scipy-openblas32>0.3.26"]
```

Reply:
I believe my change is correct. The automatic process we have for installation keeps suggesting that, but since I don't have time to pursue it, I will just change it back.